dev/PowerNotifications/PowerNotifications.cpp

```diff
@@ -0,0 +1,501 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <PowerNotifications.h>
+#include <PowerNotificationsPal.h>
+#include <PowerManager.g.cpp>
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    
+    PowerManager::eventType PowerManager::m_energySaverStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_batteryStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_powerSupplyStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_remainingChargePercentChangedEvent;
+    PowerManager::eventType PowerManager::m_remainingDischargeTimeChangedEvent;
+
+    PowerManager::eventType PowerManager::m_powerSourceStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_displayStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_systemIdleStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_powerSchemePersonalityChangedEvent;
+    PowerManager::eventType PowerManager::m_userPresenceStatusChangedEvent;
+    PowerManager::eventType PowerManager::m_systemAwayModeStatusChangedEvent;
+
+    EnergySaverStatusRegistration      PowerManager::m_energySaverStatusHandle;
+    CompositeBatteryStatusRegistration PowerManager::m_batteryStatusHandle;
+    DischargeTimeRegistration          PowerManager::m_dischargeTimeHandle;
+    PowerConditionRegistration         PowerManager::m_powerConditionHandle;
+    DisplayStatusRegistration          PowerManager::m_displayStatusHandle;
+    SystemIdleStatusRegistration       PowerManager::m_systemIdleStatusHandle;
+    PowerSchemePersonalityRegistration PowerManager::m_powerSchemePersonalityHandle;
+    UserPresenceStatusRegistration     PowerManager::m_userPresenceStatusHandle;
+    SystemAwayModeStatusRegistration   PowerManager::m_systemAwayModeStatusHandle;
+
+    std::mutex PowerManager::m_mutex;
+
+    int   PowerManager::m_batteryChargePercent, PowerManager::m_oldBatteryChargePercent;
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es10-declare-one-name-only-per-declaration - split this across two lines.